model/src/main/scala/aqua/model/ValueModel.scala

```diff
@@ -12,6 +12,8 @@ sealed trait ValueModel {
   def resolveWith(map: Map[String, ValueModel]): ValueModel = this
 
   def usesVarNames: Set[String] = Set.empty
+
+  def toRaw: ValueRaw
```

Comment:
Why is it needed? It reverses the flow of transformations: raw -> model -> res -> gen... Why does model need to become a raw value again?